packages/apollo-link-ws/README.md

```diff
@@ -32,5 +35,12 @@ WS Link takes either a subscription client or an object with three options on it
 
 By default, this link uses the [subscriptions-transport-ws](https://github.com/apollographql/subscriptions-transport-ws) library for the transport.
 
+### options
+
+an object with the following keys
+
+- `connectionCallback`: callback for connection state changes
+- `requeried`: a function passed the operation and returns a truth value indicating if it should be refetched on web socket reconnect

```

Comment:
`requeried` seems like an odd key here? Can you reword this? I'm not sure I follow

Reply:
done that it is now requeryOnReconnect